OpenDuetPkgDefines.fdf.inc

```diff
@@ -0,0 +1,102 @@
+## @file
```

Comment:
I wonder if it would be better to rename this file to just OpenDuetDefines.fdf, no additional .inc extension? This is purely for syntax highlighting purpose. E.g. In VS Code using the walonli/edk2-vscode extension, then this file is not syntax highlighted with its current name, but is with the suggested name change.

Reply:
Okay, apologies, I looked in UDK/EDK-II and you are using standard naming - so I guess the issue here is more about a PR to the VC Code extension I mentioned. Thanks!

Reply:
https://github.com/WalonLi/edk2-vscode/issues/17#issuecomment-1630656894